catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/CustomizableField.view.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+define([
+        'icanhaz',
+        'marionette',
+        'backbone',
+        'wreqr',
+        'underscore',
+        'jquery',
+        'js/view/Field.view.js',
+        'text!templates/customFieldList.handlebars'
+    ],
+    function (ich, Marionette, Backbone, wreqr, _, $, Field, customList) {
+
+        ich.addTemplate('customList', customList);
+
+        var Customizable = {};
+
+        Customizable.CustomizableCollectionView = Marionette.Layout.extend({
+            template: 'customList',
+            regions: {
+                customFieldsRegion: '#custom-fields'
+            },
+            events: {
+                "click .add-custom-field": 'addField'
+            },
+            initialize: function () {
+                this.listenTo(wreqr.vent, 'removeField:' + this.model.get('segmentId'), this.removeField);
+            },
+            onRender: function(){
+                var customFields = [];
+                var allFields = this.model.get('fields').models;
+                _.each(allFields, function (field) {
```

Comment:
This `_.each` feels like a filter. Something like:

```javascript
var customFields = this.model.get('fields')
  .filter(function (field) {
    return field.get('custom') !== undefined;
  });
```

might be more idiomatic.